data/plugins/skill/magic/magic.rb

```diff
@@ -101,6 +104,8 @@ def execute
     if @pulses == 0
       id = @item.id
 
+      mob.send(DISPLAY_SPELLBOOK)
```

Comment:
thank you for pulling this behavior up into the `execute` method. I can't think of any spells that do not return to the spellbook tab after casting :+1: